agent/policyMgr/manager.go

```diff
@@ -91,6 +104,13 @@ func (a *policyManager) ManagePolicy(payload fleet.AgentPolicyRPCPayload) {
 		} else {
 			// attempt to apply the policy to the backend. status of policy application (running/failed) is maintained there.
 			be := backend.GetBackend(payload.Backend)
+			if updatePolicy {
```

Comment:
again, this code is backend diagnostic so we shouldn't mention pktvisor or force pktvisor semantics.